react-native-zendesk-support.podspec

```diff
@@ -12,16 +12,36 @@ Pod::Spec.new do |s|
   s.source = { :git => 'https://github.com/shiftsmartinc/react-native-zendesk-support.git',  :tag => 'v'+s.version.to_s }
   s.homepage = 'https://github.com/shiftsmartinc/react-native-zendesk-support'
   s.source_files   = 'ios/RNZenDeskSupport.{h,m}'
+  s.static_framework = true
 
-  s.platform = :ios, "8.0"
+  s.platform = :ios, "10"
```

Comment:
Should this be `10.0` or is this fine

Reply:
yeah 10 is still an incredibly old target version, 8 though is mad outdated